proto/streamlit/proto/ForwardMsg.proto

```diff
@@ -116,6 +116,11 @@ message ForwardMsgMetadata {
 
   // DEPRECATED: This is not used anymore.
   ElementDimensionSpec element_dimension_spec = 3;
+
+  // active_script_hash the forward message is associated from.
+  // For multipage apps v1, this will always be the script running
+  // For multipage apps v2, this can be the main script or the page script
```

Comment:
isn't the statement "this will always be the script running" also true for v2? `active_script` is always the code that is executing right now, no matter whether its in the main file or in `page.run(...)`

Reply:
Yea. That's a better statement! Thanks for helping me clarify.